fuzzit.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+set -xe
+
+# Validate arguments
+if [ "$#" -ne 1 ]; then
+    echo "Usage: $0 <fuzz-type>"
+    exit 1
+fi
+if [ -z "$FUZZIT_API_KEY" ]; then
+    echo "Set FUZZIT_API_KEY to your Fuzzit API key"
+    exit 2
+fi
+
+# Configure
+NAME=ffjson
+TYPE=$1
+
+# Setup
+make
+(
+    cd tests/fuzz
+    rm -f target_ffjson.go
+    ffjson target.go
+)
+export GO111MODULE="off"
+go get -u github.com/dvyukov/go-fuzz/go-fuzz github.com/dvyukov/go-fuzz/go-fuzz-build
```

Comment:
I don't love using `go get` like this to install go-fuzz, or the `./...` bellow this.  Is there a way to get a validated release of these tools or a base docker image we can more cleanly depend upon?

Reply:
Will investigate this.